hack/jenkins/minikube_cross_build_and_upload.sh

```diff
@@ -36,7 +36,7 @@ set -e
 gsutil cp gs://minikube-builds/logs/index.html gs://minikube-builds/logs/${ghprbPullId}/index.html
 
 # Exit if the cross build failed.
-if [ "$?"-ne 0]; then echo "cross build failed"; exit 1; fi
+if [ "$?"-ne 0 ]; then echo "cross build failed"; exit 1; fi
```

Comment:
I had a compilation error in an earlier revision that I couldn't find without this fix.

Reply:
ah oops i suppose that space is important